airbyte-webapp/src/pages/connections/ConnectionsRoutes.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { Suspense } from "react";
+import { Navigate, Route, Routes } from "react-router-dom";
+
+import { LoadingPage } from "components";
+
+import { RoutePaths } from "../routePaths";
+import { AllConnectionsPage } from "./AllConnectionsPage";
+import { ConnectionPage } from "./ConnectionPage/ConnectionPage";
+import { ConnectionReplicationPage } from "./ConnectionReplicationPage";
+import { ConnectionSettingsPage } from "./ConnectionSettingsPage";
+import { ConnectionStatusPage } from "./ConnectionStatusPage";
+import { ConnectionTransformationPage } from "./ConnectionTransformationPage";
+import { CreateConnectionPage } from "./CreateConnectionPage/CreateConnectionPage";
+import { ConnectionPageRoutePaths } from "./types";
+
+export const ConnectionsRoutes: React.FC = () => (
+  <Suspense fallback={<LoadingPage />}>
+    <Routes>
+      <Route path={RoutePaths.ConnectionNew} element={<CreateConnectionPage />} />
+      <Route path={ConnectionPageRoutePaths.ROOT} element={<ConnectionPage />}>
+        <Route path={ConnectionPageRoutePaths.STATUS} element={<ConnectionStatusPage />} />
+        <Route path={ConnectionPageRoutePaths.REPLICATION} element={<ConnectionReplicationPage />} />
+        <Route path={ConnectionPageRoutePaths.TRANSFORMATION} element={<ConnectionTransformationPage />} />
+        <Route path={ConnectionPageRoutePaths.SETTINGS} element={<ConnectionSettingsPage />} />
+        <Route index element={<Navigate to={ConnectionPageRoutePaths.STATUS} replace />} />
+      </Route>
+      <Route index element={<AllConnectionsPage />} />
+    </Routes>
+  </Suspense>
+);
```

Comment:
Could we move these routes to `routes.tsx` and `cloudRoutes.tsx` respectively? It's a little duplication across those two files, but I think it would be nice to have the whole route tree visible in one file. It would also match what I did for the destination paths.

Reply:
IMO it's not a bad thing that developers be aware that there are two separate routers for cloud and OSS 😀 especially since they are not exactly the same. But since this branch of routes is identical, this approach is also fine for me. I will go ahead and refactor the destination routes in a similar fashion to this so that we have a consistent pattern of sub-routes.